CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
-* None.
+* Add `@ThreadSafe` property wrapper. This property wrapper type may be passed between threads.
```

Comment:
This doesn't really tell me what I'd use it for. It should mention that you use it on local variables and function parameters to make it so that the value can be used on multiple threads.